TTHAnalysis/macros/verifyAllFTrees.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+MODULE="leptonJetReCleanerSusyEWK2L"
+BASEPATH="/nfs/fanae/user/vischia/TREES_80X_011216_Spring16MVA_skim_2lep_OR_1lep_2tau/"
```

Comment:
Not sure what this script is good for at all since it anway calls the verifyFTree with the same arguments as passed to this script. Do we really need it in the repository? If you just wanna save the default values, you could keep it in a local script outside of the repo.